code/game/objects/items/weapons/storage/holster.dm

```diff
@@ -0,0 +1,266 @@
+		/obj/item/gun/projectile/gyropistol,
+		/obj/item/gun/projectile/handmade_pistol,
+		/obj/item/gun/projectile/flare_gun,
+		/obj/item/gun/projectile/lamia,
+		/obj/item/gun/projectile/mk58,
+		/obj/item/gun/projectile/olivaw,
+		/obj/item/gun/projectile/mandella,
+		/obj/item/gun/energy/gun,
+		/obj/item/gun/energy/chameleon,
+		/obj/item/gun/energy/captain,
+		/obj/item/gun/energy/stunrevolver,
+		/obj/item/gun/projectile/revolver,
+		/obj/item/gun/projectile/automatic/molly,
+		/obj/item/gun/projectile/paco,
+		/obj/item/gun/projectile/shotgun/doublebarrel/sawn, //short enough to fit in
+		/obj/item/gun/launcher/syringe,
+		/obj/item/gun/energy/plasma/brigador,
+		/obj/item/gun/projectile/shotgun/pump/sawn,
+		/obj/item/gun/projectile/boltgun/obrez,
+		/obj/item/gun/energy/retro/sawn,
+		/obj/item/gun/projectile/automatic/luty,
+		// /obj/item/gun/projectile/revolver/hornet
+		)
+
+	sliding_behavior = TRUE
+
+/obj/item/storage/pouch/holster/baton
+	name = "baton sheath"
+	desc = "Can hold a baton, or indeed most weapon shafts."
+	icon_state = "baton_holster"
+	item_state = "baton_holster"
+	rarity_value = 50
+
+	storage_slots = 1
+	max_w_class = ITEM_SIZE_BULKY
+
+	can_hold = list(
+		/obj/item/melee,
+		/obj/item/tool/crowbar,
+		/obj/item/tool/hammer/,
+		/obj/item/tool/hatchet
+		)
+
+	sliding_behavior = TRUE
+
+/obj/item/storage/pouch/holster/belt
+	name = "belt holster"
+	desc = "Can hold two handguns in. Quick on the draw!"
+	icon_state = "belt_holster"
+	item_state = "belt_holster"
+	rarity_value = 40
+
+	slot_flags = SLOT_BELT | SLOT_DENYPOCKET
+	price_tag = 240
+	matter = list(MATERIAL_BIOMATTER = 5)
+
+	sound_in = 'sound/effects/holsterin.ogg'
+	sound_out = 'sound/effects/holsterout.ogg'
+
+	max_w_class = ITEM_SIZE_HUGE
+
+	storage_slots = 2
+
+//Sheath
+/obj/item/storage/pouch/holster/belt/sheath
+	name = "sheath"
+	desc = "A sturdy brown leather sheath with a gold trim, made to house a variety of swords."
+	icon = 'icons/obj/sheath.dmi'
+	icon_state = "sheath_0"
+	item_state = "sheath_0"
+
+	price_tag = 120
+
+	storage_slots = 1
+
+	sound_in = 'sound/effects/sheathin.ogg'
+	sound_out = 'sound/effects/sheathout.ogg'
+
+	can_hold = list(
+		/obj/item/tool/sword/nt,
+		/obj/item/tool/sword/nt_sword,
+		/obj/item/tool/sword/saber,
+		/obj/item/tool/sword/katana,
+		/obj/item/tool/sword/katana/nano,
+		/obj/item/tool/sword,
+		/obj/item/toy/katana,
+		/obj/item/tool/sword/improvised
+		)
+	cant_hold = list(
+		/obj/item/tool/knife/dagger/nt,
+		/obj/item/tool/sword/nt/halberd,
+		/obj/item/tool/sword/nt/spear
+		)
+
+/obj/item/storage/pouch/holster/belt/sheath/improvised
+	name = "makeshift sheath"
+	desc = "A sturdy metal sheath with a rough finish. There's killing to do, draw your junkblade."
+	icon_state = "sheath_scrapsword"
+	price_tag = 50
+	matter = list(MATERIAL_STEEL = 2, MATERIAL_PLASTIC = 1)
+	can_hold = list(/obj/item/tool/sword/improvised)
+
+//Accessory holsters
+/obj/item/clothing/accessory/holster
```

Comment:
Return the variation. There realy is no fucking reason to remove those nice-looking, even if old, different sprites.

Reply:
Nice-looking, lmao. There is two low effort sprites, with two holster types sharing one (they differ in name only).

Reply:
"Nice-looking"? They look staggeringly ugly, I am going to just resprite one of them then. Since it's a concealed carry holster, it shouldn't really have much look to it, else how can it be concealed?